### ISSUE TITLE
roc-toolkit fails to build on aarch64 with libunwind

### ISSUE THREAD

**Issue by yarda** [roc-streaming/roc-toolkit#481]
roc-toolkit uses C++98 standard, but recent changes to libunwind are C++11 which causes roc-toolkit build to fail with error:
```
In file included from /usr/include/libunwind.h:5,
                 from src/modules/roc_core/target_libunwind/roc_core/backtrace.cpp:10:
/usr/include/libunwind-aarch64.h:198:17: error: expected identifier before numeric constant
  198 |         alignas(16) uint8_t __reserved[(66 * 8)];
      |                 ^~
/usr/include/libunwind-aarch64.h:198:17: error: expected ‘,’ or ‘...’ before numeric constant
/usr/include/libunwind-aarch64.h:198:19: error: expected ‘;’ at end of member declaration
  198 |         alignas(16) uint8_t __reserved[(66 * 8)];
      |                   ^
      |                    ;
```
It's due to the following libunwind upstream change:
https://github.com/libunwind/libunwind/pull/186/commits/e72fe5376e7084017d8f56ecdfea3d4766e5e3fb

It seems that just dropping the explicit C++98 requirement from the scons SConstruct file and compiling with the Fedora (f38) default C++11 did the trick.

**Comment by yarda** [comment]
libunwind PR: https://github.com/libunwind/libunwind/pull/186

**Comment by gavv** [comment]
Hi, thanks for report.

This is now fixed in ff4bcd430f36fdaf9ffe28336c7affd3cf6ca466 and included into v0.2.0.

Feel free to reopen if you encounter any problems.

**Comment by gavv** [comment]
See also #472
